plugin_interpreter/src/rethink_interface.py

```diff
@@ -279,18 +282,18 @@ def create_plugin_table(self, plugin_data):
             the plugin and the list of Commands (plguin_name, command_list)
         """
 
-        if verify(plugin_data[1], Commands()):
-            self._create_table("Plugins", plugin_data[0])
+        if verify(plugin_data, Commands()):
+            self._create_table("Plugins", plugin_name)
             try:
-                rethinkdb.db("Plugins").table(plugin_data[0]).insert(
-                    plugin_data[1],
+                rethinkdb.db("Plugins").table(plugin_name).insert(
```

Comment:
if you're updating anyway, thoughts on using `brain.queries.writes.create_plugin` function followed by `brain.queries.writes.advertise_plugin_commands` function